code/datums/datumvars.dm

```diff
@@ -139,6 +139,8 @@
 	var/list/names = list()
 	if (!islist)
 		for (var/V in D.vars)
+			if(V == "vars" || V == "var_edited")
+				continue

```

Comment:
Please do not add snowflake to VV code, i worked hard to remove it.

add this to vv_get_var or what ever its called.